### DISCUSSION TITLE
Capitation Question

### DISCUSSION THREAD

**Discussion by angelamarie0975** [CMSgov/price-transparency-guide#298] [Q&A]
@tdfow I have a question about the last schema update that was made to the INN example for Capitation. BillingCode/Type/Description was added into the in-network object, but how is this going to work for capitation arrangements? For example - if the billing code stays at this level, then how does that work with the covered_services object? Typically for capitation, the one capitated PMPM rate would include multiple covered_services, which explains the need for that object, but that seems redundant if we are going to list out all of the codes separately in the in_network object. 

Can you explain in more detail the intent of the Capitation arrangements and what exactly we should be including for this in the MRF? Because my understanding of Capitation doesn't seem to fit with the schema and sample provided. It just seems like there would be a lot of redundant data in the Capitation file because we would be including the billing codes in 2 spots, when it really only applies to the covered services. If there are other rates for procedures outside of the covered services, then those should be represented by other negotiated arrangement types within the file for the same provider/tin.

I laid out an example of the way that I have interpreted the file to look if we added both the billing_codes from the covered_services object. Am I interpreting this the wrong way? If not, doesn't this seem like it would be a lot of redundancy in data if we keep the schema this way for Capitation?

Thoughts anyone??

"in_network": [{ 
    "negotiation_arrangement": "capitation",
    "name": "Primary Care Capitation",
    "description": "Typical items and services for a primary care provider",
    "billing_code_type": "CPT",
    "billing_code_type_version": "2020",
    "billing_code": "27447",
    "negotiated_rates": [{
      "provider_groups": [{
        "npi": [1111111111, 2222222222, 3333333333, 4444444444, 5555555555],
         "tin":{
           "type": "ein",
           "value": "11-1111111"
         },
{ 
    "negotiation_arrangement": "capitation",
    "name": "Primary Care Capitation",
    "description": "Typical items and services for a primary care provider",
    "billing_code_type": "CPT",
    "billing_code_type_version": "2020",
    "billing_code": "27446",
    "negotiated_rates": [{
      "provider_groups": [{
        "npi": [1111111111, 2222222222, 3333333333, 4444444444, 5555555555],
         "tin":{
           "type": "ein",
           "value": "11-1111111"
         }],
    "covered_services": [{
      "billing_code_type": "CPT",
      "billing_code_type_version": "2020",
      "billing_code": "27447",
      "description": "Under Repair, Revision, and/or Reconstruction Procedures on the Femur (Thigh Region) and Knee Joint"
    },{
      "billing_code_type": "CPT",
      "billing_code_type_version": "2020",
      "billing_code": "27446",
      "description": "Under Repair, Revision, and/or Reconstruction Procedures on the Femur (Thigh Region) and Knee Joint"
    }]
  }]
}

**Comment by sharma-TIP** [comment]
Great question. We are having the same thought. And since Covered Services is an optional object. We have removed it from the Capitation schema. If the schema have to make more sense then either the Covered Services object needs to be required and service codes under In-Network object array optional.

**Comment by tdfow** [comment]
Based on how the schema was defined, the example didn't have the required billing code information, so to me either the schema would need to change or the example.   
For when the negotiation arrangement is capitation, we were were planning to use a LOCAL billing code type to describe the type of capitation, for example if the capitation arrangement was for Lab Services, then use a LOCAL billing code value would identify the capitation arrangement then report all the items/services covered in covered_services object.

**Reply by sharma-TIP** [reply]
@tdfow Thanks! That's a great idea of using a Local billing code and use the covered services object. For us it is easier to not use Covered service object just use all the valid service codes and repeat for all provider references.

**Reply by angelamarie0975** [reply]
@tdfow That's not a bad idea actually - We were originally just going to use the description attribute to specify what type of capitation arrangement it was but that was before they added the billing_code and billing_code_type to this object for capitation. But if we followed your idea, that would mean we would have to set up some type of static table to store these new LOCAL billing codes - at least, that's my initial thought process. How is your organization planning to implement that? 

However, does anyone think this would be confusing to a consumer if they are trying to go by this schema when they consume the file to provide whatever analytics? I guess it's the same as us creating our other 'LOCAL' codes though, so this might actually work - just means more logic we have to implement. 

So are you suggesting the schema end up looking like this?
  "in_network": [{
  "negotiation_arrangement": "capitation",
  "name": "PMPM AGE 0-18",
  "description": "PMPM AGE 0-18 - Typical items and services for a primary care provider",
  "billing_code_type": "LOCAL",
  "billing_code_type_version": "2021",
  "billing_code": "1234567890",
  "negotiated_rates": [{"provider_groups": [{"npi": [1111111111, 2222222222, 3333333333, 4444444444, 5555555555],
                                                                     "tin":{"type": "ein",
                                                                      "value": "11-1111111"}
	                                                           ],
                                    "negotiated_prices": [{"negotiated_type": "negotiated",
                                                                       "negotiated_rate": 25000.00,
                                                                       "expiration_date": "2022-01-01",
                                                                       "service_code": ["05", "06", "07"],
                                                                       "billing_class": "professional"}
                                                                     ]
                                  }],
  "covered_services": [{"billing_code_type": "CPT",
                                    "billing_code_type_version": "2021",
                                    "billing_code": "27447",
                                    "description": "Under Repair, Revision, and/or Reconstruction Procedures on the Femur (Thigh Region) and Knee 
                                      Joint"},
                                  {"billing_code_type": "CPT",
                                   "billing_code_type_version": "2021",
                                   "billing_code": "27446",
                                   "description": "Under Repair, Revision, and/or Reconstruction Procedures on the Femur (Thigh Region) and Knee 
                                   Joint"}
                                  ]
  },{
  "negotiation_arrangement": "capitation",
  "name": "PMPM AGE 19+",
  "description": "PMPM AGE 19+ - Typical items and services for a primary care provider",
  "billing_code_type": "LOCAL",
  "billing_code_type_version": "2021",
  "billing_code": "0987654321",
  "negotiated_rates": [{"provider_groups": [{"npi": [666666666, 7777777777],
                                                                     "tin":{"type": "ein",
                                                                      "value": "11-1111111"}
	                                                           ],
                                    "negotiated_prices": [{"negotiated_type": "negotiated",
                                                                       "negotiated_rate": 25000.00,
                                                                       "expiration_date": "2022-01-01",
                                                                       "service_code": ["05", "06", "07"],
                                                                       "billing_class": "professional"}
                                                                     ]
                                  }],
  "covered_services": [{"billing_code_type": "CPT",
                                    "billing_code_type_version": "2021",
                                    "billing_code": "27447",
                                    "description": "Under Repair, Revision, and/or Reconstruction Procedures on the Femur (Thigh Region) and Knee 
                                      Joint"},
                                  {"billing_code_type": "CPT",
                                   "billing_code_type_version": "2021",
                                   "billing_code": "27446",
                                   "description": "Under Repair, Revision, and/or Reconstruction Procedures on the Femur (Thigh Region) and Knee 
                                   Joint"}
                                  ]
  }
]

**Reply by tdfow** [reply]
@angelamarie0975  Yes, we are supporting these as LOCAL codes.   We also have to support some other customized negotiated rate scenarios that are LOCAL (ffs) along with the capitation scenarios.  The covered_services would contain all the items/services that are associated with the capitation where the capitation negotiated arrangement is the LOCAL code that defines the capitation that is being addressed in the covered_services object for the negotiated rate.

**Reply by angelamarie0975** [reply]
@tdfow so - after having some conversations within our organization and our parent organization, it was decided that if the PMPM rates were in any part related to member specific demographic data, that it would not be applicable to the MRF - that we only need to include the PMPM rates if we have a standard base rate not affected by a member at all, especially since we have an underlying fee schedule. So if we do have capitated rates that fit that scenario, we will report them and I am going to have our team use the solution you stated above as that is the one that makes the most sense to me. 

However - I have a couple of questions for you:
1. For the covered_services object - are you just including EVERY possible billing_code? We have our underlying fee schedules that I was going to have included in this object, but if there is no underlying schedule, would you just list every single possible HCPCS, CPT, DRG, etc?
2. You stated that you have other negotiated rate scenarios along with the capitation scenarios. Do you mean ffs that are capitated at a max amount or just other types of rates that you will be disclosing as 'LOCAL'. The reason I am asking is that for some of our % of Billed rates, the allowed amount is capped at a certain amount. In my mind, that could be a capitated arrangement and would warrant 2 records in the file - one for the % of Billed and the other for the Capitated rate. I asked this in another discussion 403 - https://github.com/CMSgov/price-transparency-guide/discussions/403, but I am curious what your thoughts are on this. Does your organization have any similar contracts and if so, how you are handling those? If we don't have a way to report the capped amount, then a contract that is say .38 of charge and the charge is 10000, the allowed amount is 3800 - however, it is capped at 1200. We shouldn't report the 1200 as the allowed amount though because if the .38 % is only 800, then 1200 is not correct - but we also don't have any way to let the consumer know that this is what would happen since we can't report that the procedure is capped. The only thing I can think of is to include that as a capitated arrangement as stated above. I was only thinking about capitation in the respects of like PMPM rates - until we got the answers about % of billed and we are now implementing that part, I didn't think about the capped rates for those provider contracts.

**Reply by tdfow** [reply]
@angelamarie0975  Regarding the covered_service object ---yes, this is a HUGE object of billing codes.   I do think with the introduction of the new billing code value of CSTM-00, if all services for a given type type (CPT, HCPCS, etc) apply--then I think this could be used if it is applicable to the situation.
 "billing_code_type": "CPT",
 "billing_code_type_version": "2020",
 "billing_code": "**CSTM-00**",
 "description": "All CPT codes",
Regarding 2.   We have a few different scenarios, some are handled with LOCAL codes, while there are situations where there are services with ffs arrangements that  are also included in the covered_services object for a capitation arrangement.

**Comment by NigelEllisBSC** [comment]
As a payor we are having this conversation as well.  From what we understand there is no requirement to show our PMPM rates in MRF.  Is that everyone's understanding?  We could show our negotiated fees that could vary depending on age banding as shown above, but those may not be the rates the IPA is truly using and could be misleading. Thanks.

**Reply by tdfow** [reply]
@NigelEllisBSC The mandate has specific references for PMPM and other capitation models --- for example page 257 ( The Departments acknowledge that there are many different types of capitation models. As stated in the example earlier, for capitation arrangements that reimburse a provider a capitated amount, such as a PMPM, or a similar direct primary care arrangement, the plan or issuer would report the negotiated rate, which in this case is the PMPM amount, and the underlying fee schedule, as applicable.)

**Reply by NigelEllisBSC** [reply]
So we would list all the procedures that are covered by the IPA/provider and the rate for them would be the monthly PMPM we pay?  If we are reporting an IPA then do we only need to include the IPA NPI in the provider_groups and not all the providers who work for the IPA? Is there a more efficient way to do this as it seems like a lot of duplication and we may not know all of the procedure codes provided by the IPA.  How does reporting the PMPM rate help a consumer of the MRF determine the lowest cost of care in a given area?

**Reply by tdfow** [reply]
@NigelEllisBSC I am just providing a reference within the Transparency in Coverage document that addresses some guidelines/requirements around PMPM rates and other types of capitation.   There are a few pages that talk about different capitation scenarios and how to address based on the capitation model of the Plan.
The covered_services object would be used to report the codes that are included in the capitation arrangement.

**Reply by RLTx1391** [reply]
@NigelEllisBSC how did you guys address IPAs?
